README.md

```diff
@@ -69,7 +71,7 @@ import pkg/loony
 
 type AnyRefObject = ref object
 
-var loonyQueue = initLoonyQueue[AnyRefObject]()
+var loonyQueue = newLoonyQueue[AnyRefObject]()
```

Comment:
Wait, what?  Why does it have to be an `object`?  Or at least, this is how the code reads...

Reply:
Could you clarify this?